cmd/sblookup/main.go

```diff
@@ -39,8 +39,9 @@ import (
 )
 
 var (
-	apiKeyFlag   = flag.String("apikey", "", "specify your Safe Browsing API key")
-	databaseFlag = flag.String("db", "", "path to the Safe Browsing database. By default persistent storage is disabled (not recommended).")
+	apiKeyFlag    = flag.String("apikey", "", "specify your Safe Browsing API key")
+	databaseFlag  = flag.String("db", "", "path to the Safe Browsing database. By default persistent storage is disabled (not recommended).")
+	serverUrlFlag = flag.String("server", safebrowsing.DefaultServerURL, "Safebrowsing API server address")
```

Comment:
Nit: rename to serverURLFlag (Go style is to use all caps for abbreviations)
Nit: Add period to end of description.